### ISSUE TITLE
Prompts panel: Save button for prompt name not accessible by keyboard and missing accessible name

### ISSUE THREAD

**Issue by kangabell** [danny-avila/LibreChat#5535]
There is a button with a floppy disk icon on it that is unlabeled and I cannot tab to it with a keyboard.

## WCAG Criteria
2.5.3 Label in Name - A	

## Testing Method
Manual 

## Screenshot
![floppy disk icon to the right of text box](https://github.com/user-attachments/assets/98edf16d-04a8-40fc-af9d-b13bab86c3b1)

## Code Snippet
```<button type="button" class="rounded p-2 hover:bg-gray-300/50 dark:hover:bg-gray-700"><svg viewBox="64 64 896 896" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round" class="icon-md" width="1.2em" height="1.2em" fill="currentColor" xmlns="http://www.w3.org/2000/svg">...</svg></button>```

## Recommended Action
Don't hide the save button until the user has exited the entire form area, not just the text input. Separating the edit button from the save button so they are unique elements may simplify this process. Also, add button text or an aria-label that says "save name" or similar.
